tests/test_core_network.py

```diff
@@ -3017,3 +3042,65 @@ def test_getting_link_slope_dictionary(network3):
     link_slope = (z_2 - z_1) / length
     slope_dict = network3.get_link_slope_dictionary(elevation_dictionary)
     assert slope_dict['0']['slope'] == link_slope
+
+
+def test_generating_summary_report(network5):
+    network5.schedule = Schedule(epsg='epsg:27700', services=[
```

Comment:
just write that schedule in the fixture, atm that fixture uses `correct_schedule` and then overwrites it in the test